### DISCUSSION TITLE
[BUG] When trying to access the Mirth channel I am facing a Java Heap space issue

### DISCUSSION THREAD

**Discussion by harvindernanda** [nextgenhealthcare/connect#6344] [Q&A]
While I can access the Mirth dashboard successfully I am unable to access the Mirth channel 

Issue seen as below:
java.lang.OutOfMemoryError: Java heap space
com.mirth.connect.client.core.ClientException: java.lang.OutOfMemoryError: Java heap space
at com.mirth.connect.client.core.Client$2.invoke(Client.java:294)
at com.sun.proxy.$Proxy66.getChannelSummary(Unknown Source)
at com.mirth.connect.client.core.Client.getChannelSummary(Client.java:1063)
at com.mirth.connect.client.ui.ChannelPanel.retrieveChannels(ChannelPanel.java:614)
at com.mirth.connect.client.ui.ChannelPanel.retrieveChannels(ChannelPanel.java:608)
at com.mirth.connect.client.ui.ChannelPanel$1.doInBackground(ChannelPanel.java:429)
at com.mirth.connect.client.ui.ChannelPanel$1.doInBackground(ChannelPanel.java:426)
at com.mirth.connect.client.ui.QueuingSwingWorker.doInBackground(QueuingSwingWorker.java:97)

We have tried re-starting the servers as well as increased the heap space to 4GB at the time of launch too but this did not help

**Comment by jonbartels** [comment]
This should be a Discussion not an Issue.

2-4GB on the client should be more than enough.
<img width="1033" alt="image" src="https://github.com/user-attachments/assets/2e081152-c94e-4a93-a11f-eb08696f485d">

When you get the client error you posted, do you also see a server side error in `mirth.log`?

**Reply by pacmano1** [reply]
Converted to discussion.

**Comment by harvindernanda** [comment]
There are no server side errors seen in mirth.log. Also please note in the organisation at the moment I am the only user who is experiencing this issue. The other users can access the channels successfully.

**Reply by jonbartels** [reply]
This issue sounds familiar but I cannot think of what to search for.

What OS and Launcher version are you using?

Some ideas:

1. Does the channel you are loading have lots of tags?
2. Delete the MC launcher app from your desktop and re-install
3. https://github.com/nextgenhealthcare/connect/discussions/5843 describes a similar issue and says how to launch the launcher from the command line to see more log output

**Reply by harvindernanda** [reply]
Launcher version is 1.3.1 running on Windows 10. I would have thought it could be channels or multiple tags but it runs fine for the other users and only causes an issue for me.

**Reply by pacmano1** [reply]
@harvindernanda given it happens to you and only you, I am not sure you will get an answer from this community.  @jonbartels suggested reasonable actions.   For deleting the launcher and reinstalling it, you would want to make sure to remove its program directory.   Also, you did not share you Windows 10 available memory, another possible problem.   You did not share the connection profile either.  The less you share, the harder it is to help you.
